content/introduction/getting-started.md

```diff
@@ -473,10 +472,10 @@ $ HANAMI_ENV=test bundle exec hanami db prepare
 
 ### Working With Entities
 
-An entity is something really close to a plain Ruby object.
+An entity is something really close to an everyday Ruby object.
```

Comment:
```suggestion
An entity is something really close to a plain Ruby object.
```
I think the word `plain` is better here, since "Plain old Ruby object" is [a common term](https://codesthq.com/hi-im-poro/), and often abbreviated as "PORO" (though I don't think we shoud introduce that acronym here).

And it comes from Java, where it's also [a common term](https://en.wikipedia.org/wiki/Plain_old_Java_object)

Reply:
Why do you want to avoid using PORO? That would have clarified it for me a bit.